src/domain/profile/modal/EditProfileModal.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { Formik, FieldArray, FormikErrors } from 'formik';
+import { useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+
+import styles from './editProfileModal.module.scss';
+import { ProfileType } from '../type/ProfileTypes';
+import { registrationFormDataSelector } from '../../registration/state/RegistrationSelectors';
+import { defaultRegistrationData } from '../../registration/state/RegistrationReducers';
+import { Guardian } from '../../guardian/types/GuardianTypes';
+import Modal from '../../../common/components/modal/Modal';
+import EnhancedInputField from '../../../common/components/form/fields/input/EnhancedInputField';
+import InputField from '../../../common/components/form/fields/input/InputField';
+import SelectField from '../../../common/components/form/fields/select/SelectField';
+import { SUPPORT_LANGUAGES } from '../../../common/translation/TranslationConstants';
+import Button from '../../../common/components/button/Button';
+
+export type EditProfileModalValues = {
+  firstName: string;
+  lastName: string;
+  email: string | null;
```

Comment:
i dont think email is nullable.. let ts destroy us

Reply:
I have already changed the typing, so this is resolved. yay.